.github/workflows/docker-publish.yml

```
@@ -0,0 +1,65 @@
+  REGISTRY: docker.io
+  IMAGE_NAME: quegenx/vercel-mcp-server
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+    - name: Checkout repository
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 4

(indentation)